api/tests/routes/backoffice/venues_test.py

```diff
@@ -1262,6 +1263,35 @@ def test_update_venue_siret_disabled(self, client, roles_with_permissions, offer
         assert venue.publicName == "Ma boutique"
         assert venue.siret == original_siret
 
+    def test_update_venue_ban_id(self, authenticated_client):
+        # When
+        bo_user = users_factories.AdminFactory()
+        backoffice_api.upsert_roles(bo_user, [perm_models.Roles.SUPPORT_PRO])
+
+        venue = offerers_factories.VenueFactory()
+
+        data = self._get_current_data(venue)
+        data["is_permanent"] = True
+        data["ban_id"] = "15152_0024_00003"
+
+        # Given
```

Comment:
```suggestion
        # When
```
Mais je les ai enlevés dans les tests BO, tu peux donc retirer ces commentaires

Reply:
c'était par habitude , j'enlève

Reply:
Ça ne gêne pas vraiment, sauf si c'est à l'envers :)